skycoin/skywire
# Attach started by updater binaries to TTY of their parent

Did you run `make format && make check`? Not yet

Fixes #336 

 Changes:	
- Attach started by updater binaries to TTY of their parent

How to test this PR:
- Run the integration env
- Run the following:
    - macOS: `curl -k -d '{"checksums_url":"https://github.com/nkryuchkov/skywire/releases/download/v0.0.1-alpha.test.tty/checksums.txt", "archive_url":"https://github.com/nkryuchkov/skywire/releases/download/v0.0.1-alpha.test.tty/skywire-v0.0.1-alpha.test.tty-darwin-amd64.tar.gz", "version": "v0.0.1-alpha.test.tty"}' -X POST https://localhost:8000/api/visors/024ec47420176680816e0406250e7156465e4531f5b26057c9f6297bb0303558c7/update
`
    - Linux/amd64: `curl -k -d '{"checksums_url":"https://github.com/nkryuchkov/skywire/releases/download/v0.0.1-alpha.test.tty/checksums.txt", "archive_url":"https://github.com/nkryuchkov/skywire/releases/download/v0.0.1-alpha.test.tty/skywire-v0.0.1-alpha.test.tty-linux-amd64.tar.gz", "version": "v0.0.1-alpha.test.tty"}' -X POST https://localhost:8000/api/visors/024ec47420176680816e0406250e7156465e4531f5b26057c9f6297bb0303558c7/update
`
- The updating of `VisorA` should finish with no errors and an updated binary should be started
- Press Ctrl-C, the updated binary should be stopped
